siteapp/settings_application.py

```diff
@@ -19,27 +19,58 @@
 
     'loadtesting',
 ]
-OKTA_CONFIG = environment.get("okta", {})
+OIDC_CONFIG = environment.get("oidc", {})
+
+# default endpoint values for different OpenID Connect providers
+OIDC_PROFILES = {
+    'okta': {
+        "oidc_op_jwks_endpoint": "/oauth2/v1/keys",
+        "oidc_op_authorization_endpoint": "/oauth2/v1/authorize",
+        "oidc_op_token_endpoint": "/oauth2/v1/token",
+        "oidc_op_user_endpoint": "/oauth2/v1/userinfo",
+    },
+    'auth0': {
+        "oidc_op_jwks_endpoint": "/.well-known/jwks.json",
+        "oidc_op_authorization_endpoint": "/authorize",
+        "oidc_op_token_endpoint": "/oauth/token",
+        "oidc_op_user_endpoint": "/userinfo",
+    }
+}
 # https://blog.theodo.com/2021/03/okta-sso-with-django-admin/ - example for login override
-if OKTA_CONFIG:
+if OIDC_CONFIG:
     LOGIN_ENABLED = False
-    AUTHENTICATION_BACKENDS += ['siteapp.authentication.OIDCAuthentication.OIDCAuth', ]
-
-    OKTA_DOMAIN = OKTA_CONFIG['domain']
-    BASE_URL = environment['govready-url'].replace(':443', '')
+    AUTHENTICATION_BACKENDS += ['siteapp.authentication.OIDCAuthentication.OIDCAuth', ]   
+    OIDC_DOMAIN = OIDC_CONFIG['domain']
+    BASE_URL = environment['govready-url'].replace(':443', '')  
```

Comment:
Should we change this to use `blueprint-url`?

Reply:
yes, but it will be require cascading changes.  we should ticket this work if we haven't already.